builtin/providers/aws/data_source_aws_ecs_cluster_test.go

```diff
@@ -0,0 +1,35 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSEcsDataSource_ecsCluster(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckAwsEcsClusterDataSourceConfig,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.aws_ecs_cluster.default", "status", "ACTIVE"),
+					resource.TestCheckResourceAttr("data.aws_ecs_cluster.default", "pending_tasks_count", "0"),
+					resource.TestCheckResourceAttr("data.aws_ecs_cluster.default", "running_tasks_count", "0"),
+					resource.TestCheckResourceAttr("data.aws_ecs_cluster.default", "registered_container_instances_count", "0"),
+				),
+			},
+		},
+	})
+}
+
+const testAccCheckAwsEcsClusterDataSourceConfig = `
```

Comment:
Mind specifying the name with a random integer, in the off-chance there's a leak during the test? 
```
func testAccCheckAwsEcsClusterDataSourceConfig(rInt int) string {
  return fmt.Sprintf(`resource "aws_ecs_cluster" "default" {
  name = "tfacctest-%d"
}
data "aws_ecs_cluster" "default" {
  cluster_name = "${aws_ecs_cluster.default.name}"
}`, rInt)
}
```